src/scss/cozy.scss

```diff
@@ -126,4 +126,10 @@ app-vault-ciphers {
             } 
         }
     }
+}
+
+#login-page {
+    .error-text {
+        color: #FFD3D3;
```

Comment:
CSS var please

Reply:
Fixed. Current CSS variable is `#FECDC6` instead of `#FFD3D3` which is pretty similar.

Correct value comes with Cozy-UI v55 but upgrade seems to be more difficult than I thought : https://github.com/cozy/cozy-pass-web/issues/71